storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -127,7 +127,9 @@ public static void main(String[] args) throws Exception {
         Worker worker = new Worker(conf, null, stormId, assignmentId, Integer.parseInt(supervisorPort),
                                    Integer.parseInt(portStr), workerId);
         worker.start();
-        Utils.addShutdownHookWithForceKillIn1Sec(worker::shutdown);
+        LOG.info("Adding shutdown hook with kill in {} secs", ObjectReader.getInt(conf.get(Config.SUPERVISOR_WORKER_SHUTDOWN_SLEEP_SECS)));
```

Comment:
`ObjectReader.getInt(conf.get(Config.SUPERVISOR_WORKER_SHUTDOWN_SLEEP_SECS))` will be executed twice once end users configure to print INFO or lower. Would be better to have separate variable here.

Reply:
Good catch, will fix